tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
In addition, it might be better to use

    fonts.names.font_file_lookup('greciliae.ttf')

(untested). This is defined [in luaotfload](https://github.com/lualatex/luaotfload/blob/master/src/luaotfload-database.lua).